src/core/control/XournalMain.cpp

```diff
@@ -490,6 +490,7 @@ void on_startup(GApplication* application, XMPtr app_data) {
     app_data->gladePath = std::make_unique<GladeSearchpath>();
     initResourcePath(app_data->gladePath.get(), "ui/about.glade");
     initResourcePath(app_data->gladePath.get(), "ui/xournalpp.css", false);
+    initResourcePath(app_data->gladePath.get(), "resources-templates/toolbar.ini", false);
```

Comment:
That doesn't work on MacOS. The file is installed in the`ui` folder. So did you mean this?
```suggestion
    initResourcePath(app_data->gladePath.get(), "ui/toolbar.ini", false);
```

Reply:
It's not found after a build, since it's generated into resources-templates. So we either must leave it or we must change the generation directory.

Reply:
@bhennion and I have agreed to always `install` before running Xournal++. See https://github.com/xournalpp/xournalpp/pull/5226#issuecomment-1781444487

Reply:
Unfortunately that won't integrate well with current toolchains and build flows. The build directory should already be in a state, that we can just copy parts of it into the install directory.

Reply:
@rolandlo @bhennion, we should add something like this to the CMake scripts, to ensure that everyone uses the installation:

```
if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
    set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "Default install path." FORCE)
    set_target_properties(xournalpp PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/install/bin")
endif()

add_custom_command(
    TARGET xournalpp POST_BUILD
    COMMAND ${CMAKE_COMMAND} --install ${CMAKE_BINARY_DIR} --config $<CONFIG>
)
```

This will automatically install Xournal++ into <build>/install/bin.
It also integrates with build tools, since the `RUNTIME_OUTPUT_DIRECTORY` is the same as the installation's `RUNTIME DESTINATION` directory

Reply:
With `install/bin` you mean `<build_folder>/install/bin`, right?

I'm not familiar with `RUNTIME_OUTPUT_DIRECTORY` and would have to try on different operating systems, but your suggestion seems reasonable to me in case it works as expected.

Reply:
On windows it works as expected, but this builds directly into the <builddir>/install/bin directory. I would like to have it built into the builddir and then copied. But then it's not recognized anymore by vscode-cmake-tools.

Reply:
Will resolve as out of scope and created a new issue.